lib/locales/it.ts

```diff
@@ -408,23 +397,43 @@ export const locale: Locale = {
-      'Santa Teresa Benedetta della Croce, vergine, martire e patrona secondaria d’Europa',
+    teresa_benedicta_of_the_cross_stein_virgin_copatroness_of_europe: 'Santa Teresa Benedetta della Croce, vergine, martire e patrona secondaria d’Europa',
     teresa_of_jesus_of_avila_virgin: 'Santa Teresa di Gesù d’Avila, vergine e dottore della Chiesa',
-    therese_of_the_child_jesus_and_the_holy_face_of_lisieux_virgin:
-      'Santa Teresa di Gesu’ Bambino, vergine e dottore della Chiesa',
+    therese_of_the_child_jesus_and_the_holy_face_of_lisieux_virgin: 'Santa Teresa di Gesu’ Bambino, vergine e dottore della Chiesa',
     thomas_apostle: 'San Tommaso, apostolo',
     thomas_aquinas_priest: 'San Tommaso d’Aquino, sacerdote e dottore della Chiesa',
     thomas_becket_bishop: 'San Tommaso Becket, vescovo e martire',
+    thursday_of_the_lord_s_supper: 'Giovedì Santo / Cena del Signore',
     timothy_of_ephesus_and_titus_of_crete_bishops: 'Santi Timoteo e Tito, vescovi',
+    transfiguration: 'Trasfigurazione del Signore',
+    trinity_sunday: 'Santissima Trinitá',
     turibius_of_mogrovejo_bishop: 'San Turibio de Mogrovejo, vescovo',
     vincent_de_paul_priest: 'San Vincenzo de’ Paoli, sacerdote',
     vincent_ferrer_priest: 'San Vincenzo Ferrer, sacerdote',
     vincent_of_saragossa_deacon: 'San Vincenzo, diacono e martire',
     visitation_of_mary: 'Visitazione della Beata Vergine Maria',
     wenceslaus_i_of_bohemia_martyr: 'San Venceslao, martire',
     wenceslaus_i_of_bohemia_martyr_patron_of_the_czech_nation: 'San Venceslao, martire e patrono della nazione ceca',
+    adelaide_of_burgundy_empress: 'Santa Adelaide di Borgogna, imperatrice',
+    adelphus_of_metz_bishop: 'Sant’Adelfo di Metz, vescovo',
+    all_saints_of_the_archdiocese_of_paris: 'Tutti i Santi dell’Arcidiocesi di Parigi',
+    all_saints_of_the_diocese_of_saint_denis: 'Tutti i Santi della Diocesi di Saint-Denis',
+    amarin_of_alsace_abbot: 'Sant’Amarino, abate e martire',
+    arbogast_of_strasbourg_bishop: 'Sant’Arbogasto, vescovo',
+    arbogast_of_strasbourg_bishop_patron_of_the_archdiocese_of_strasbourg: 'Sant’Arbogasto, vescovo e patrono dell’Arcidiocesi di Strasburgo',
+    amandus_of_strasbourg_bishop: 'Sant’Amando di Strasburgo, vescovo',
+    aurelia_of_strasbourg_virgin: 'Sant’Aurèlia di Strasburgo, vergine',
+    benedict_of_aniane_abbot: 'San Benedetto d’Aniane, abate',
+    blessed_martyrs_of_paris: 'Beati Martiri della Rivoluzione francese',
+    brigid_of_kildare_virgin: 'Saint Brigida di Kildare, badessa',
+    brigid_of_kildare_virgin_copatroness_of_ireland: 'Saint Brigida di Kildare, badessa e and patrona secondaria d’Irlanda',
+    carmelites_of_compiegne_virgins_and_martyrs: 'Beati carmelitani di Compiègne, vergini e martiri',
+    catherine_zoe_laboure_virgin: 'Santa Caterina Labourè, vergine',
+    ceraunus_of_paris_bishop: 'San Cerauno di Parigi, vescovo',
+    charles_of_jesus_de_foucauld: 'Beato Carlo de Foucauld, sacerdote',
```

Comment:
@gtortone, what is more used in Italian: `Carlo de Foucauld` or `Carlo di Gesù`?

Reply:
in Italy it "Charles de Foucauld" is very common... there are a lot of non-profit associations with this name... I've never heard "Carlo de Foucauld"

Reply:
Okay, I’ve just found [this website](https://www.charlesdefoucauld.it/en/?start=10). I’ll change his name back to `Charles`. Thanks! :pray: